doc/nrf/ug_radio_fem.rst

```diff
@@ -0,0 +1,178 @@
+.. _ug_radio_fem:
+
+Radio Front-end Module (FEM) support
+####################################
+
+|NCS| allows you to extend the radio range of your board with an implementation of the Front-End Modules (FEM).
+The FEM support is based on the :ref:`nrfxlib:mpsl_fem`, which is integrated in nrfxlib's MPSL library.
+This library provides nRF21540 GPIO and Simple GPIO implementations, for 3-pin and 2-pin PA/LNA interfaces, respectively.
+
+This guide describes how to add support for these FEM implementations to your application in |NCS|:
+
+* :ref:`ug_radio_fem_nrf21540_gpio` - For the nRF21540 GPIO implementation that uses nRF21540.
+* :ref:`ug_radio_fem_skyworks` - For the Simple GPIO implementation that uses the SKY66112-11 device.
+
+These methods are only available to protocol drivers that are using FEM features provided by the MPSL library in multiprotocol scenarios.
+They are also valid for cases where an application uses just one protocol, but benefits from features provided by MPSL.
+To avoid conflicts, check the protocol documentation to see if it uses FEM support provided by MPSL.
```

Comment:
Shall we add a note here that currently none of protocols delivered by Nordic uses FEM support provided by MPSL, but it may be used by radio protocols implemented by customers?

Reply:
@czeslawmakarski , I think we briefly discussed it today. Do you have any comments?

Reply:
I think some note like 'Currently no protocols support it' shall be enough.

Reply:
I added such a note

Reply:
This note will be obsolete after next upmerge or hal_nordic revision update :)